src/components/main/MainScreen.js

```diff
@@ -45,7 +47,9 @@ const MainScreen = () => {
       headerLeftContent={leftContent}
       headerRightContent={<HeaderRightContent id={mainId} />}
     >
-      {content}
+      <LayoutContextProvider>
+        <SideContent item={item}>{[content]}</SideContent>
```

Comment:
I'm not sure it's good to override the usual type of children. Isn't it possible to use a component instead of an array?